### PR TITLE
[*] BO : Module Page - Improved Ajax Loading on catalog

### PR DESCRIPTION
**Pull request by tchauviere**

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php [hunk 1]
```diff
@@ -106,7 +76,42 @@ public function refreshCatalogAction(Request $request)
...
+    {
+        $formattedContent = [];
+        $formattedContent['selector'] = '.module-catalog-page';
+        $formattedContent['content'] = $this->render(
+            'PrestaShopBundle:Admin/Module/Includes:sorting.html.twig',
+            [
+                'totalModules' => count($products)
+            ]
+        )->getContent();
+        $formattedContent['content'] .=$this->render(
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php]: space missing (I'm always surprised how PHP can handle this kind of stuff ahaha =)

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<div class="dropdown">
+    {% for menu in topMenuData %}
+        <button class="btn dropdown-toggle module-categories-dropdown" id="{{menu.refMenu}}" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig]: you better use a variable:

```twig
{% set refMenu = menu.refMenu %}
```

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_status.html.twig [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<div class="dropdown">
+    <button class="btn dropdown-toggle module-status-dropdown" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        <div class="module-status-selector-label">
+            Show all modules
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_status.html.twig]: Does theses strings need to be translated ?

File: src/PrestaShopBundle/Resources/views/Admin/Module/catalog.html.twig [hunk 4]
```diff
@@ -17,16 +17,16 @@
...
-        <div class="col-md-10 col-md-offset-1">
+        <div class="col-md-10 col-md-offset-1 module-catalog-page">
             {# Addons connect modal #}
             {% include 'PrestaShopBundle:Admin/Module/Includes:modal_addons_connect.html.twig' %}
             {# Contains toolbar-nav for module page #}
             {% include 'PrestaShopBundle:Admin/Module/Includes:modal_import.html.twig' %}
             {# Contains menu with Selection/Categories/Popular and Tag Search #}
-            {% include 'PrestaShopBundle:Admin/Module/Includes:menu_top.html.twig' with { 'topMenuData': topMenuData } %}
+            {% include 'PrestaShopBundle:Admin/Module/Includes:menu_top.html.twig' %}
             {# Temporary loader until ajax has finisehd and display the full catalog #}
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Module/catalog.html.twig]: has finished*

File: src/PrestaShopBundle/Resources/views/Admin/Module/catalog.html.twig [hunk 5]
```diff
@@ -17,16 +17,16 @@
...
             {% include 'PrestaShopBundle:Admin/Module/Includes:grid_loader.html.twig' %}
-            {# Module Categories Grid #}
+            {# Module Categories Grid @TODO move out from here #}
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Module/catalog.html.twig]: do it or dont, but please dont add @Todo, it's better to create a tech story with what you expect to be done here :heart:

**Comment by maximebiloe** [comment]
@tchauviere, could you update your PR according to @mickaelandrieu's feedback, please?

**Pull request merged**